prihlas-me.php

```diff
@@ -15,8 +15,23 @@
 
     echo "Toto je ".$_SESSION['pokusu_prihlaseni'].". pokus o přihlášení.";
 
-    $uzivatelskeJmeno = $_GET['jmeno'];
+    $uzivatelskeJmeno = strtolower($_GET['jmeno']); // u jména nechceme řešit velikost písmen
     $uzivatelskeHeslo = $_GET['heslo'];
+
+    // kontrola prihlaseni
+    foreach ($admini as $admin) {
```

Comment:
Nemůžeme tohle zjednodušit na něco snáze pochopitelného, jako jenom ` $_GET['jmeno'] == "login" && $_GET['heslo'] == "bflmpsvz"`? Tohle vubec neni jednoduchy pochopit co dela... 

Nebo to nechat jako bonus, ale mozna bych tech vic uctu uplne vyhodil, jinak se na tom ztrati dost casu. A mit proste spravne jen jedmu kombinaci jmena a hesla a cus.